otelcollector/prometheusreceiver/metrics_receiver.go

```diff
@@ -251,6 +254,9 @@ func (r *pReceiver) applyCfg(cfg *config.Config) error {
 	if err := r.discoveryManager.ApplyConfig(discoveryCfg); err != nil {
 		return err
 	}
+
+	r.webHandler.ApplyConfig(cfg)
```

Comment:
chk with Grace on what this change is

Reply:
Thanks Rashmi, this change is so that the target UX config is updated to correspond with the config the target allocator gives since it updates in place